AutoMatrix/AutoMatrix.py

```diff
@@ -0,0 +1,837 @@
+
+import logging
+import os,sys,time,zipfile,urllib.request,shutil
+
+import vtk
+
+import slicer
+from slicer.ScriptedLoadableModule import *
+from slicer.util import VTKObservationMixin
+
+import qt
+import glob
+import numpy as np
+from qt import QFileDialog,QMessageBox
+from functools import partial
+import SimpleITK as sitk
+
+from Matrix_CLI.Apply_matrix_utils.GZ_tools import GetPatients
+from Matrix_CLI.Apply_matrix_utils.VTK_tools import GetPatientsVTK
```

Comment:
Since Matrix_CLI is a module, that directory is not available at runtime when installed via the extension index; instead the contents are all installed to the same directory. See below the directory structure when installed:

![image](https://github.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools/assets/10702931/8f162013-0cc9-4a74-bd82-7e6dc555cc22)

Therefore these two imports fail; instead use

```py
from Apply_matrix_utils.GZ_tools import GetPatients
from Apply_matrix_utils.VTK_tools import GetPatientsVTK
```

Reply:
I remove the folder Matrix_CLI but thank you I will now it for the next time !